package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zoralabs/nft-components",
-  "version": "0.1.12",
+  "version": "0.1.15",
```

Comment:
nit: can this package change be removed so the version bump can be done independently of PRs

Reply:
Sure.

Reply:
Done @iainnash

Reply:
Looks like something went wrong here – can you try `git fetch origin main && git checkout origin/main -- package.json CHANGELOG`

Reply:
Done @iainnash